lua/nvim-web-devicons/icons-default.lua

```diff
@@ -155,6 +155,18 @@ local icons_by_filename = {
     cterm_color = "113",
     name = "Zshrc",
   },
+  [".zsh_plugins.txt"] = {
+    icon = "󱉋",
+    color = "#ff0288",
+    cterm_color = "198",
+    name = "Antidote",
+  },
+  [".zsh_history"] = {
+    icon = "",
+    color = "#6d8086",
+    cterm_color = "66",
+    name = "History",
+  },
```

Comment:
```suggestion
```

This isn't standard either, but result of setting

```
       HISTFILE
              The file to save the history in when an interactive shell exits.  If unset, the history is not saved.
```

Please remove.

Reply:
ok